_guide/_pages/interviews/wrap-up.md

```diff
@@ -56,26 +56,37 @@ Begin by introducing yourself by saying this or something similar to it:
 
-- “Tell me about yourself”
+### Relevant work experience
 
-- ["What drew you to 18F?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.6qssbuvkhie2)
+- ["Can you tell me a little about your background and how 18F fits into that picture?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.6qssbuvkhie2)
 
   - If possible, be prepared to notify the TTS Talent Team about how this candidate entered our candidate funnel should we extend an offer (this helps improve our outreach efforts).
 
-- ["Tell me about your experience with agile or other collaborative working styles. How does working in an agile way affect the technical choices you make?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.p3zgydisjuv3)
+- ["Whether or not you’ve had direct experience with agile, in an agile environment, how do you approach trade-offs?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.d4ctqn683zxs)
 
-- ["Have you had a chance to participate in user-centered design and/or user research?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
+- ["Tell me about your experience working on teams that directly involved non-engineers. How did you approach working with others with different professional skills?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.p2cr6nh5culs)
 
-- ["Have you ever worked with a remote/distributed team before?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.95oxlva5ufr7)
+- ["What has your experience been with user-centered design and/or user research?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.1407do81qrqo)
 
-- ["Please explain a complex technical concept or tool to me as if I were a non-technical partner"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.8lizmty3jisl)
+### Technology and open source
+Ask one or more of the following:
+- ["Are you comfortable with sharing your work in a public fashion?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
+- ["What is your comfort level with peer-reviewed work?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
+- ["Your product owner is very excited about a proprietary subscription based product which would cost twice what an open source solution would. How would you encourage the product owner to use open source?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.hmo4ig69x3qn)
```

Comment:
This question implies the cost isn’t worth it, but doesn’t explicitly state it. I’d probably indicate either way:

> Your product owner is very excited about a proprietary subscription-based product which would cost twice what an open source solution would with no added benefits. How would you encourage the product owner to use open source?

Or:

> Your product owner is very excited about a proprietary subscription-based product which would cost twice what an open source solution would. How would you compare the two options and help direct the product owner?

Reply:
+1 to the first option

Reply:
I like the second option –  it leaves room for all the nuance that this topic has. There are so many dimensions to consider, especially in government, other than cost (assuming that "cost" means total cost of ownership). It's a frequent activity in our engagements and evaluating a candidate's ability to unpack/prioritize those tradeoffs seems valuable.